samples/fbuffer/Connectors/SumConnector.cpp

```diff
@@ -115,8 +114,8 @@ bool SumConnector::REDISPopLogs(long long int len __attribute__((unused)), std::
 
     redis_reply = redis.Query(std::vector<std::string>{"GETSET", sum_name, "0"}, result, true);
     if (redis_reply == REDIS_REPLY_NIL) {
-        DARWIN_LOG_INFO("SumConnector:: REDISPopLogs:: key '" + sum_name + "' does not exist (yet?)");
-        return false;
+        DARWIN_LOG_DEBUG("SumConnector:: REDISPopLogs:: key '" + sum_name + "' did not exist");
+        result = std::string("0");
```

Comment:
This modification seems to be done to resolve an issue that I don't know nor can identify the issue, I can't review it

Reply:
When Redis returns a NIL, it means that the key does not exist
For our use, the key might not exist yet, so this is not an invalid read (so we mustn't return false)

However, I can't remember why I casted the value to std::string("0") here, but I will improve the code